tests/Integration/Behaviour/Features/Context/Domain/ProductFeatureContext.php

```diff
@@ -55,6 +63,158 @@ public function setProductTaxRulesGroup(string $taxRulesGroupReference, string $
         Cache::clean('product_id_tax_rules_group_*');
     }
 
+    /**
+     * @When I add product :productReference with following basic information in default language:
+     *
+     * @param string $productReference
+     * @param TableNode $table
+     */
+    public function addProduct(string $productReference, TableNode $table): void
+    {
+        $data = $table->getRowsHash();
+        $defaultLangId = (int) Configuration::get('PS_LANG_DEFAULT');
```

Comment:
I think instead of only using default language you could use https://github.com/PrestaShop/PrestaShop/blob/95c1d2c7537d71599fa373462bf885b0ebbcaa18/tests/Integration/Behaviour/Features/Context/Domain/AbstractDomainFeatureContext.php#L136

like in

https://github.com/PrestaShop/PrestaShop/blob/95c1d2c7537d71599fa373462bf885b0ebbcaa18/tests/Integration/Behaviour/Features/Context/Domain/CurrencyFeatureContext.php#L159

It is also usable to test returned localized data like in:

https://github.com/PrestaShop/PrestaShop/blob/95c1d2c7537d71599fa373462bf885b0ebbcaa18/tests/Integration/Behaviour/Features/Context/Domain/CurrencyFeatureContext.php#L231

This would allow you to add data like

```
| name  | en-US:bottle of beer |
```

It adds two advantages:
- you can check that localized values are correctly inserted
- you can test an error scenario when the default language name is not provided